CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- (GH #468) add support for project isolation for projects marked as restricted
```

Comment:
might want to rebase changelog as we changed it in v2.0.0

Reply:
Yes, a rebase would be in order, it'll come soon.